packages/shared/src/components/Markdown.tsx

```diff
@@ -1,12 +1,79 @@
-import React, { ReactElement } from 'react';
+import React, { ReactElement, useEffect, useState } from 'react';
+import ReactDOM from 'react-dom';
+import { tippy } from '@tippyjs/react';
 import { sanitize } from 'dompurify';
 import styles from './markdown.module.css';
+import { ProfileTooltipContent } from './profile/ProfileTooltipContent';
+import { profileTooltipClasses } from './profile/ProfileTooltip';
+import { useProfileTooltip } from '../hooks/useProfileTooltip';
+import { getUserPermalink } from '../lib/user';
 
-export default function Markdown({
-  content,
-}: {
+const classes = Object.values(profileTooltipClasses).join(' ');
+
+interface MarkdownProps {
   content: string;
-}): ReactElement {
+}
+
+type TippyInstance = ReturnType<typeof tippy>[0];
+
+export default function Markdown({ content }: MarkdownProps): ReactElement {
+  const [userId, setUserId] = useState('');
+  const [instance, setInstance] = useState<TippyInstance>();
```

Comment:
Why do we keep it in the state?
I see only where we assign a value to `instance` but not where we use it

Reply:
We will be using it in the very last part of generating the tooltip content. Below is line:

https://github.com/dailydotdev/apps/blob/fc87a2724404f9a9c0ef33e217fdca1c22f967f7/packages/shared/src/components/Markdown.tsx#L74